src/Ilios/CliBundle/Command/FixLearningMaterialMimeTypesCommand.php

```diff
@@ -0,0 +1,206 @@
+        
+        parent::__construct();
+    }
+    
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('ilios:maintenance:fix-mime-types')
+            ->setDescription('Cleanup incorrectly stored mime types for learning materials.');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $totalLearningMaterialsCount = $this->learningMaterialManager->getTotalLearningMaterialCount();
+
+        $helper = $this->getHelper('question');
+        $output->writeln('');
+        $question = new ConfirmationQuestion(
+            '<question>Ready to fix ' . $totalLearningMaterialsCount .
+            ' learning materials. Shall we continue? </question>' . "\n",
+            true
+        );
+        
+        if ($helper->ask($input, $output, $question)) {
+            $progress = new ProgressBar($output, $totalLearningMaterialsCount);
+            $progress->setRedrawFrequency(208);
+            $output->writeln("<info>Starting cleanup of learning materials...</info>");
+            $progress->start();
+
+            $fixed = 0;
+            $skipped = 0;
+            $offset = 0;
+            $limit = 50;
+            $errors = [];
+
+            while ($fixed + $skipped < $totalLearningMaterialsCount) {
+                /** @var LearningMaterialInterface[] $learningMaterials */
+                $learningMaterials = $this->learningMaterialManager->findBy([], ['id' => 'desc'], $limit, $offset);
+                foreach ($learningMaterials as $lm) {
+                    $mimetype = $lm->getMimetype();
+                    if ($lm->getCitation()) {
+                        if ($mimetype != 'citation') {
+                            $lm->setMimetype('citation');
+                            $this->learningMaterialManager->update($lm, false);
+                            $fixed++;
+                        } else {
+                            $skipped++;
+                        }
+                    } elseif ($lm->getLink()) {
+                        if ($mimetype != 'link') {
+                            $lm->setMimetype('link');
+                            $this->learningMaterialManager->update($lm, false);
+                            $fixed++;
+                        } else {
+                            $skipped++;
+                        }
+                    } elseif ($path = $lm->getRelativePath()) {
+                        if (in_array($mimetype, ['link', 'citation'])) {
+                            $file = $this->iliosFileSystem->getFile($path);
+                            if (false === $file) {
+                                $errors[] = 'File not found for learning material # ' . $lm->getId();
+                                $skipped++;
+                            } else {
+                                try {
```

Comment:
try/catch block should only surround `$newMimeType = $file->getMimeType();`, that's the error we're after (not any db errors that could bubble up from failed attempts to update the LM record).